README.org

```diff
@@ -55,16 +55,16 @@
       ("C-c r G" . gkroam-update-all)))
    #+END_SRC
 
-** Dependency
+** Dependencies
 
-   * [[https://github.com/nicferrier/emacs-db][db]] is used to cache page and headline id.
-   * [[https://github.com/company-mode/company-mode][company]] is used to complete double brackets link and hashtags.
-   * [[https://github.com/BurntSushi/ripgrep][ripgrep(rg)]] is used to search reference link.
+   * [[https://github.com/nicferrier/emacs-db][db]] is used to cache page and headline ids.
+   * [[https://github.com/company-mode/company-mode][company]] is used to auto-complete double bracket links and hashtags.
+   * [[https://github.com/BurntSushi/ripgrep][ripgrep(rg)]] is used to search reference links.
 
 ** Tips (important)
 
-   - Install ripgrep(rg) command line firstly and make sure it is in exec-path list.
-   - Do not set =company-idle-delay= variable to zero, it will causes an error. see [[https://github.com/company-mode/company-mode/issues/455][issue#455]].
+   - Install the command line search tool ripgrep(rg) first and make sure it is in the exec-path list.
+   - Do not set the =company-idle-delay= variable to zero, it will causes an error. See [[https://github.com/company-mode/company-mode/issues/455][issue#455]].
    - Use =gkroam-build-caches= to generate all healines' id and cache them in db manually.
```

Comment:
I wasn't exactly sure what "healines" was. Was it headings? or Headlines?

- Use gkroam-build-caches to generate all headline ids and cache them in the db manually?
- Use gkroam-build-caches to generate all heading ids and cache them in the db manually?